form-runner/jvm/src/main/resources/apps/fr/components/view.xsl

```diff
@@ -633,7 +633,7 @@
 
     <xsl:template match="fr:title" name="fr-title">
         <!-- Q: Why do we need @ref here? -->
-        <xh:h1><xf:output value="{if (exists(@ref)) then @ref else '$title'}"/></xh:h1>
+        <xh:h5 class="text-white-50"><xf:output value="{if (exists(@ref)) then @ref else '$title'}"/></xh:h5>
     </xsl:template>
```

Comment:
Keep `h1`, but add `fs-3 mb-0` classes to make smaller/center.

Reply:
Done.